### PR TITLE
75 update readme

### PR DESCRIPTION
**Pull request by acse-ej321**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,57 @@
...
+Goalie provides goal-oriented mesh adaptation pipelines for solving partial differential equations (PDEs) using adapted meshes built on the Python-based finite element library [Firedrake](http://www.firedrakeproject.org/).  It runs a fixed point iteration loop for progressively solving time-dependent PDEs and their adjoints on sequences of meshes, performing goal-oriented error estimation, and adapting the meshes in sequence with a user-provided adaptor function until defined convergence criteria have been met. It is recommended that users are familiar with adjoint methods, mesh adaptation and the goal-oriented framework before starting with Goalie.
 
-In order to use Goalie's metric-based mesh adaptation functionality, you will need a bespoke Firedrake installation.
-See the instructions for the [Animate](https://github.com/pyroteus/animate) package for details on this.
+Goal-oriented mesh adaptation presents one of the clearest examples of the intersection between adjoint methods and mesh adaptation. It is an advanced topic, so it is highly recommended that users are familiar with adjoint methods, mesh adaptation and the goal-oriented framework before starting with Goalie.
```
Comment (jwallwork23) [inline README.md:9]: This repeats the above.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,57 @@
...
+
+### To install Firedrake via shell script
+
+Firedrake, along with PETSc, is required by the Goalie package and is available for installation via a shell script.
+
+Instructions:
+- Download installation files either:
+	-  manually from: or via `curl -O <filename>`
+		- install/install_firedrake_custom_mpi.sh
+		- /install/petsc_options.txt
```
Comment (jwallwork23) [inline README.md:33]: Please remove initial `/`.

**Review by jwallwork23 — CHANGES_REQUESTED** [review]
Thanks for this! Just a couple of minor requests.

File: README.md (hunk 1 above)
Comment (acse-ej321) [reply README.md:9]: I removed the second version. I also added a link to the pyroteus.github.io where you go into goal-oriented mesh adaptation given the suggestion for the user to familiarise themselves with it.

File: README.md (hunk 2 above)
Comment (acse-ej321) [reply README.md]: done

**Review by jwallwork23 — APPROVED** [review]

**Pull request merged**